src/js/components/viewer/models/audio-media-object.jsx

```diff
@@ -41,11 +39,14 @@ var AudioMediaObject = React.createClass({
         var player = audioPlayer || this.state.player;
         return new TWEEN.Tween(position)
             .to(target, duration)
-            .onUpdate(function() {
-                if(this.volume){
-                    this.volume(position.vol);
+            .onUpdate(function(obj) {
+                try {
+                    player.volume(obj.vol);
+                } catch (e) {
+                    console.log("ERROR for onUpdate");
```

Comment:
I got a certain pattern of steps that cause the audio object to error, and a console error saying that we are setting the state to an unexisting component.
Steps: 
1.Have Media Browser and Scene viewer components open. Select  AP_BEACH_FRONT_SOUNDS from the scene selector
2.Refresh the page and the results are assumed to be automatically selected  AP_BEACH_FRONT_SOUNDS .Your components look like they have the scene loaded.
3. Go to scene list and select the none option, follow up by selecting AP_BEACH_FRONT_SOUNDS again. You should receive errors in the console